python_modules/dagster/dagster/core/definitions/run_status_sensor_definition.py

```diff
@@ -10,13 +11,20 @@
     SensorDefinition,
     SensorEvaluationContext,
     SkipReason,
+    is_context_provided,
```

Comment:
nit: I think this is not the place is_context_provided is defined

Reply:
I see it here https://github.com/dagster-io/dagster/blob/6d7034725ba8d0d1770e5c6e4c6a880c8b4c2e99/python_modules/dagster/dagster/core/definitions/sensor_definition.py#L43 is there a different place it's also defined?

Reply:
ah nvm I remember, the context checks are different for each decorated type of function.